VAMobile/src/translations/en/common.json

```diff
@@ -58,6 +58,18 @@
   "done": "Done",
   "done.a11yHint": "Exits out of the web view and navigates to the previous page",
   "done.picker.a11yHint": "confirms selection and closes the menu",
+  "downtime.CLAIMS_HISTORY_SCREEN.featureName": "Claims",
```

Comment:
This was from my attempt to make the labels translations. Wouldn't be in the final product unless we decided to make it work correctly.